e2e-tests/gef/cypress/support/commands/loadData.js

```diff
@@ -2,14 +2,11 @@
  * Mock data loader:
  * 1. Clears all the collections
  * 2. Inserts mocked data
-*/
+ */
 
 const loadData = () => {
-  // Ensure project root directory
-  cy.exec('if [ -d "dtfs2" ]; then cd dtfs2; fi');
```

Comment:
This wasn't doing anything because each `cy.exec` runs in it's own child process (i.e. doesn't affect any other `cy.exec` commands), so even if this navigated it would have no effect on the following `cy.exec` command.